config/application.rb

```diff
@@ -201,9 +200,6 @@ class Application < Rails::Application
     # Disable noisy logging of requests for assets
     config.assets.quiet = true
 
-    # Catching Invalid JSON Parse Errors with Rack Middleware
-    config.middleware.use Foreman::Middleware::CatchJsonParseErrors
```

Comment:
Why? 6.1 provides this out of the box? could you do it in separate commit and explain in the commit message why are we doing it? :)